pytorch_lightning/core/saving.py

```diff
@@ -52,7 +52,6 @@ class ModelIO(object):
     def load_from_checkpoint(
         cls,
         checkpoint_path: str,
-        *args,
```

Comment:
this is not possible to match with saved parameters as this guy does not have any name just a value...

Reply:
@jbschiratti if the init signature of model is `__init__(self, *args, my_param=12)` and the constructor does some unpacking of args, how do we pass these into the load_from_checkpoint function?
I think I understand why you want to remove it, and we could argue that it is not supported because saving hyperparameters means we need a name for it. I'm fine with the change, but just asking to double check I'm not missing the point of it.

Reply:
ok, got it.
we can load the args but it does not make sense to pass *args to this function here. there is no way to map them to the correct args.
It is still fine if the user wants to define *args in their signature. But overriding them at load is not possible for us.